src/CSS/countryzz.css

```diff
@@ -0,0 +1,116 @@
+
+* {
+  padding: 0;
+  margin: 0;
+  box-sizing: border-box;
+}
+
+.countries-list {
+  list-style: none;
+  display: flex;
+  flex-wrap: wrap;
+  margin-top: 0;
+}
+
+.countries-list li {
+  flex: 11rem;
+  padding-bottom: 1rem;
+  background-color: #97af42;
+}
+
+.countries-list li:nth-child(4n),
+.countries-list li:nth-child(4n+1) {
+  background-color: #9ab6345d;
+}
+
+.header-wrapper {
+  display: flex;
+  background-color: rgba(255, 255, 255, 0.397);
+  justify-content: center;
+  padding-top: 2rem;
+  padding-bottom: 2rem;
+}
+
+.world-map {
+  width: 11rem;
+  height: 11rem;
+  margin-left: 2rem;
+  opacity: 0.7;
+}
+
+.world-info {
+  display: flex;
+  flex-direction: column;
+
+  /* padding-top: 2rem; */
+  text-align: center;
+  color: white;
+  transform: scale(1.5);
+  padding-block: 1rem;
+}
+
+.world-cases {
+  font-weight: bolder;
+  color: #000;
+  font-family: 'lato', sans-serif;
+  padding: 4px;
+}
+
```

Comment:
- [ ] Please, kindly correct this part on your homepage:


![image](https://user-images.githubusercontent.com/57421823/152126937-d1915f0b-8af0-4912-81e1-b6e8da8e7f7b.png)

As you can see, the elements are not centered, some of them contain an overflow.  
If you do this, your homepage will look striking! 💪